src/RestrictedPython/README.rst

```diff
@@ -7,22 +7,28 @@ RestrictedPython provides a ``restricted_compile`` function that works
 like the built-in ``compile`` function, except that it allows the
 controlled and restricted execution of code:
 
-  >>> src = '''
-  ... def hello_world():
-  ...     return "Hello World!"
-  ... '''
-  >>> from RestrictedPython import compile_restricted
-  >>> code = compile_restricted(src, '<string>', 'exec')
+.. code-block:: pycon
```

Comment:
could that be a ``doctest`` derictive? it is not in sphinx context

Reply:
you mean ``.. code-block:: doctest``?

pycon also works well if you view on github. But i can of course change if you want.

Reply:
pycon is the lexer used by pygments, so just highlighting, I meant replacing ``code-block::`` with ``doctest::``